tools/packaging/kata-deploy/README.md

```diff
@@ -133,17 +133,14 @@ This image contains all the necessary artifacts for running Kata Containers, all
 from the [Kata Containers release page](https://github.com/kata-containers/runtime/releases).
 
 Host artifacts:
+* `cloud-hypervisor`, `firecracker`, `qemu-system-x86_64`, `qemu-virtiofs-system-x86_64` and supporting binaries
+* `containerd-shim-kata-v2`
+* `kata-collect-data.sh`
 * `kata-runtime`
-* `kata-fc`
-* `kata-qemu`
-* `kata-proxy`
-* `kata-shim`
-* `firecracker`
-* `qemu-system-x86_64` and supporting binaries
 
 Virtual Machine artifacts:
-* `kata-containers.img`: pulled from Kata GitHub releases page
-* `vmlinuz.container`: pulled from Kata GitHub releases page
+* `kata-containers.img` and `kata-containers-initrd.img`: pulled from Kata GitHub releases page
+* `vmlinuz.container` and `vmlinuz-virtiofs.container`: pulled from Kata GitHub releases page
```

Comment:
Do we carry virtiofs specific patches anymore? I thought it was all upstream and we could drop?

Reply:
@egernst, I noticed this and when I did I opened https://github.com/kata-containers/kata-containers/pull/1423 and https://github.com/kata-containers/kata-containers/pull/1434.  I just don want to mix changes is this series.